modelexpress_client/python/modelexpress/refit_receiver.py

```
@@ -0,0 +1,372 @@
+
+        if min_step is None:
+            min_step = self._current_step + 1
+
+        deadline = time.perf_counter() + timeout_seconds
+
+        while True:
+            try:
+                response = self._client.list_sources(
+                    status_filter=status_filter,
+                )
+            except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)